x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/queries/TextExpansionQueryBuilderTests.java

```diff
@@ -43,11 +43,19 @@
 public class TextExpansionQueryBuilderTests extends AbstractQueryTestCase<TextExpansionQueryBuilder> {
 
     private static final String RANK_FEATURES_FIELD = "rank";
-    private static int NUM_TOKENS = 10;
+    private static final int NUM_TOKENS = 10;
 
     @Override
     protected TextExpansionQueryBuilder doCreateTestQueryBuilder() {
-        var builder = new TextExpansionQueryBuilder(RANK_FEATURES_FIELD, randomAlphaOfLength(4), randomAlphaOfLength(4));
+        TokenPruningConfig tokenPruningConfig = rarely()
+            ? new TokenPruningConfig(randomIntBetween(1, 100), randomFloat(), randomBoolean())
+            : null;
```

Comment:
This shouldn't be rarely, just make it `randomBoolean()`
And, we should have a separate test for `TokenPruningConfigTests` that extends `AbstractXContentSerializingTestCase`, That test case can provide a static method for returning a random instance.